test/integration/version.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const chai = require('chai');//assertions used
+const server = require('../../');//for  index.js file
+const pkg = require('../../package.json');
+const baseUrl="https://localhost/api/version";
+
+// Set should property of all objects for BDD assertions
+chai.should();
+
+const myObj = {
+      name: 'testObj'
+};
+
+//.eql to compare objects
+
+describe('Version', () => {
+
+    // Always wait for the server to be ready before beginning any tests
+    before('wait for server to be ready', () => {
+        return server.app.pluginsRegistered.catch((error) =>{
+        	console.dir(error);
```

Comment:
By catching this error, Mocha will never know that this task failed. If you need to catch the error for debugging purposes, please re-throw it:
```
return server.app.pluginsRegistered.catch((error) =>{
    console.dir(error);
    throw error;
});
```